components/usage/cmd/run.go

```diff
@@ -30,19 +32,39 @@ func run() *cobra.Command {
 		Run: func(cmd *cobra.Command, args []string) {
 			log.Init(ServiceName, Version, true, verbose)
 
-			err := server.Start(server.Config{
-				ControllerSchedule:    schedule,
-				StripeCredentialsFile: apiKeyFile,
-			})
+			cfg, err := parseConfig(configPath)
+			if err != nil {
+				log.WithError(err).Fatal("Failed to get config. Did you specify --config correctly?")
+			}
+
+			err = server.Start(cfg)
 			if err != nil {
 				log.WithError(err).Fatal("Failed to start usage server.")
 			}
 		},
 	}
 
+	localConfig := path.Join(os.ExpandEnv("GOMOD"), "..", "config.json")
```

Comment:
What's the use case here? Why do we expect `config.json` one level up from the `go.mod`?

Reply:
`GOMOD` points to where the `go.mod` is defined, in our case that is `GOMOD="/workspace/gitpod/components/usage/go.mod"`. The `..` drops the filename and we then join it with our desired filename.

See [playground](https://goplay.tools/snippet/pQae5ju9a88)